build.gradle

```diff
@@ -1,43 +1,36 @@
-/*#######################################################
- *
- *   Maintained by Gregor Santner, 2017-
- *   https://gsantner.net/
- *
- *   License of this file: Apache 2.0 (Commercial upon request)
- *     https://www.apache.org/licenses/LICENSE-2.0
- *
-#########################################################*/
-import java.text.SimpleDateFormat
-
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
+import java.text.SimpleDateFormat
+
 buildscript {
-    ext.version_setup_compileSdk = 28
-    ext.version_setup_minSdk = 17
-    ext.version_setup_targetSdk = ext.version_setup_compileSdk
-    ext.version_setup_buildTools =  ext.version_setup_compileSdk + ".0.3" // Specifying optional
+    ext {
+        version_plugin_kotlin = "1.2.71"
+        enable_plugin_kotlin = false
 
-    // https://developer.android.com/studio/releases/gradle-plugin.html
-    ext.version_gradle_tools = "3.2.0"
-    // https://developer.android.com/topic/libraries/support-library/revisions.html
-    ext.version_library_appcompat =  ext.version_setup_compileSdk + ".0.0"
-    // https://github.com/JakeWharton/butterknife/releases
-    ext.version_library_butterknife = "8.8.1"
-    // https://github.com/guardianproject/NetCipher/releases
-    ext.version_library_netcipher = "2.0.0-alpha1"
-    // https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-gradle-plugin#LookAtCentral
-    ext.version_plugin_kotlin = "1.2.21"
-    ext.enable_plugin_kotlin = false
+        version_compileSdk = 28
+        version_buildTools = version_compileSdk + ".0.3"
+        version_minSdk = 14 //17
```

Comment:
Keep 17

Reply:
?? But it can support ICS...? Can it not?

Reply:
On global view, I want to be on API17 (when viewing the root build.gradle) across all my projects. In MemeTastic case I lower override it to 14 in app/build.gradle.

This all magic is really just to make me maintainence really easy and in current way I just update root-gradle in one project and do file-copy to all others, and they are updated then wihout requireing project changes usually.

Reply:
ah ok then